ouroboros-consensus/changelog.d/scriv.ini

```diff
@@ -0,0 +1,13 @@
+[scriv]
+format = md
+insert_marker = Changelog entries
+md_header_level = 2
+version = literal: ../ouroboros-consensus/ouroboros-consensus.cabal: version
+categories = Patch, Non-Breaking, Breaking
+end_marker = scriv-end-here
+fragment_directory = ../ouroboros-consensus/changelog.d
+ghrel_template = {{body}}
+main_branches = master, main, develop
+new_fragment_template = file: new_fragment.${config:format}.j2
+output_file = ../ouroboros-consensus/CHANGELOG.${config:format}
+skip_fragments = README.*
```

Comment:
The default file names created by scriv seem a bit verbose, e.g.
```
20230125_134526_alexander.esgen_implement_release_process.md
```
I guess one can just rename them; but it would be even nicer if we could tweak the template, but I don't see anything related on the [Config docs page](https://scriv.readthedocs.io/en/1.2.0/configuration.html).

Reply:
I'm not sure that option exists no, I couldn't find it either